core/org/commcare/core/process/CommCareInstanceInitializer.java

```diff
@@ -28,9 +29,11 @@
     protected final CommCareSession session;
     protected CaseInstanceTreeElement casebase;
     protected LedgerInstanceTreeElement stockbase;
+    protected CacheTable<String, TreeElement> fixtureBases = new CacheTable();
```

Comment:
To get ride of lint warnings: 
`private final CacheTable<String, TreeElement> fixtureBases = new CacheTable<>();`